sdk/search/Azure.Search.Documents/src/Models/SuggestResults{T}.cs

```diff
@@ -37,8 +38,12 @@ public partial class SuggestResults<T>
         /// <summary>
         /// The sequence of suggestions returned by the query.
         /// </summary>
-        public IList<SearchSuggestion<T>> Results { get; internal set; } =
-            new List<SearchSuggestion<T>>();
+        internal List<SearchSuggestion<T>> _results = new List<SearchSuggestion<T>>();
+
+        /// <summary>
+        /// The sequence of suggestions returned by the query.
+        /// </summary>
+        public IReadOnlyList<SearchSuggestion<T>> Results => _results;
```

Comment:
Nit: this would still be castable to a mutable list, but I know we've done it this way elsewhere. Perhaps, though, you might consider wrapping `_results` in a `ReadOnlyCollection` that prevents this.

Reply:
It touches slightly more code, but it's a fair suggestion.